doc/rst/source/explain_-J_full.rst_

```diff
@@ -273,12 +273,12 @@ projection names stands for Equal-Area and Conformal, respectively):
 
 .. _-Jp_full:
 
-    **-Jp**\ [**a**]\ *scale*\ [*/origin*][**+r**\|\ **z**] or **-JP**\ [**a**]\ *width*\ [*/origin*][**+r**\|\ **z**]
+    **-Jp**\ *scale*\ [**+a**]\ [**+o**\ *origin*][**+r**\|\ **z**] or **-JP**\ *width*\ [**+a**]\ [**+o**\ *origin*][**+r**\|\ **z**]
     (Polar coordinates (*theta*, *r*))
 
-    Optionally insert **a** after **-Jp** [or **-JP**] if *theta* is azimuths CW
+    Optionally add **+a** if *theta* is azimuths CW
     from North instead of directions CCW from East [Default]. Optionally
-    append /*origin* in degrees to indicate an angular offset [0].
+    append **+o**\ *origin* in degrees to indicate an angular offset [0].
```

Comment:
I'm often confused about the definition of "offset". It seems the offset is CW by default, and CCW for **+a**. Usually I have to try both positive and negative values to find the one I need. It would be better if the description can be more clear if possible.

Reply:
OK, relearning which way it goes and will update the docs shortly.